quazip/quazip_qt_compat.h

```diff
@@ -103,11 +103,31 @@ inline QString quazip_symlink_target(const QFileInfo &fi) {
 }
 #endif
 
+// deprecation
+#if QT_VERSION >= QT_VERSION_CHECK(6, 8, 0)
+#include <QtCore/QTimeZone>
+inline QDateTime quazip_since_epoch() {
+    return QDateTime(QDate(1970, 1, 1), QTime(0, 0), QTimeZone(Qt::UTC));
+}
+
+inline QDateTime quazip_since_epoch_ntfs() {
+    return QDateTime(QDate(1601, 1, 1), QTime(0, 0), QTimeZone(Qt::UTC));
+}
+#else
+inline QDateTime quazip_since_epoch() {
+    return QDateTime(QDate(1970, 1, 1), QTime(0, 0), Qt::UTC);
+}
+
+inline QDateTime quazip_since_epoch_ntfs() {
+    return QDateTime(QDate(1601, 1, 1), QTime(0, 0), Qt::UTC);
+}
+#endif
+
 // this is not a deprecation but an improvement, for a change
```

Comment:
> // this is not a deprecation but an improvement, for a change

It looks like something that is no longer relevant

or perhaps line 137 should also be replaced with
quazip_since_epoch_ntfs()